api/v1/openapi.yaml

```diff
@@ -3506,6 +3506,15 @@ definitions:
         type: array
         items:
           "$ref": "#/definitions/BgpPeerFamilies"
+      connect-retry-time:
+        description: Initial value for the BGP ConnectRetryTimer (RFC 4271, Section 8) in seconds
+        type: integer
+      hold-time:
+        description: Initial value for the BGP HoldTimer (RFC 4271, Section 4.2) in seconds
```

Comment:
We should maybe add that this is negotiated time and not really configured time.